eclair-front/src/main/scala/fr/acinq/eclair/Boot.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 ACINQ SAS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.acinq.eclair
+
+import java.io.File
+
+import akka.actor.ActorSystem
+import com.typesafe.config.{ConfigFactory, ConfigParseOptions, ConfigSyntax}
+import grizzled.slf4j.Logging
+import kamon.Kamon
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success}
+
+/**
+ * Created by PM on 25/01/2016.
+ */
+object Boot extends App with Logging {
+  try {
+    val datadir = new File(System.getProperty("eclair.datadir", System.getProperty("user.home") + "/.eclair"))
+    val config = ConfigFactory.parseString(
+      Option(System.getenv("AKKA_CONF")).getOrElse("").replace(";", "\n"),
+      ConfigParseOptions.defaults().setSyntax(ConfigSyntax.PROPERTIES))
+      .withFallback(ConfigFactory.load())
+
+    implicit val system: ActorSystem = ActorSystem("eclair-node", config)
```

Comment:
```suggestion
    implicit val system: ActorSystem = ActorSystem("eclair-front", config)
```

Reply:
Okay this is embarrassing. I may have done this on purpose to enable grouping of actors in the monitoring...

Reply:
Yeah, I think keeping `eclair-node` is good. WDYT?

Reply:
Sounds good, a comment would be useful in that case to explain that choice

Reply:
Okay, can't have a different actorsystem name across the cluster apparently. Tried, didn't work.

https://github.com/akka/akka/blob/1bf012837cf7f0120acb61e6a83eb66a93b23ab0/akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala#L696-L700